docs/MatrixOne/Reference/Functions-and-Operators/Mathematical/uuid.md

```diff
@@ -1,53 +0,0 @@
-# **UUID()**
```

Comment:
it looks like this function is still removed?

Reply:
https://github.com/matrixorigin/matrixorigin.io/pull/440